projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -119,6 +123,32 @@ const useUpdateWebviewVariable = (
     return valueInWebview
 }
 
+const injectJavascript = (
+    webviewRef: React.MutableRefObject<WebView | null>,
+    script: string,
+): void => {
+    if (webviewRef && webviewRef.current) {
+        webviewRef.current.injectJavaScript(script)
+    }
+}
+
+/**
+ * Sometimes the webUrl is empty due to that content not being published at the point the edition
+ * was created. However, we still have access to the article path at the time of publication
+ * This function checks to see if theguardian.com/<path at publication time> exists
+ */
+const pathLookup = async (articlePath: string) => {
```

Comment:
What about the name of this function? Should we make it more explicit - checkURLExists or similar?